linkml_model/model/schema/aliases.yaml

```diff
@@ -0,0 +1,84 @@
+id: https://w3id.org/linkml/mappings
+title: LinkML Schema Mappings
+name: mappings
+description: LinkML model for mappings
+license: https://creativecommons.org/publicdomain/zero/1.0/
+version: 2.0.0
+
+prefixes:
+  linkml: https://w3id.org/linkml/
+  skos: http://www.w3.org/2004/02/skos/core#
+  OIO: http://www.geneontology.org/formats/oboInOwl#
+  IAO: http://purl.obolibrary.org/obo/IAO_
+
+default_curi_maps:
+    - semweb_context
+
+default_prefix: linkml
+default_range: string
+
+emit_prefixes:
+  - linkml
+  - rdf
+  - rdfs
+  - xsd
+  - skos
+  - OIO
+  - IAO
+
+imports:
+  - linkml:types
+  - linkml:meta
```

Comment:
so modularization is quite tricky here, I think meta will want to import this, rather than vice versa.. I don't know if there are reciprocal dependencies

but to keep it simple, maybe the alias model can just live in meta